transformations.html

```diff
@@ -249,8 +282,8 @@ <h2>Records</h2>
   , y = point.y + 1
 }
 
-dist {x,y} =                  -- pattern matching on fields
-  sqrt (x^2 + y^2)
+dist { x, y } =                  -- pattern matching on fields
```

Comment:
<img width="407" alt="Screenshot 2021-08-08 at 9 35 46 AM" src="https://user-images.githubusercontent.com/473/128617954-f6a97b4f-27d3-43e4-a45a-260373f4e8b2.png">

```suggestion
dist { x, y } =               -- pattern matching on fields
```

we could arrange comments on top of code, otherwise we'd need to constantly watch for this